releasePipeline/README.md

```diff
@@ -73,31 +73,50 @@ All the tests must pass before moving on.
+   3. Tagging approvers with a link to the regression test results.
+
+Have a look at the GHE issues for previous releases for examples on how this has been done before. 
 
 ### Deployment
 
 <!-- Commenting out the steps below for now as they do not work. An item is open to fix this. Temporary steps to work around this below: -->
 <!-- 1. Amend 30-deploy-maven-galasa.yaml and amend the version parameter to the release.
 1. Run `kubectl -n galasa-build create -f 30-deploy-maven-galasa.yaml` - Deploy the maven artifacts to OSS Sonatype. -->
-1. Pull the [galasa-obr-with-galasabld](https://harbor.galasa.dev/harbor/projects/3/repositories/galasa-obr-with-galasabld/artifacts-tab) image from Harbor.
-1. Exec into the image so you can run commands from inside it, by running `docker run -it --entrypoint /bin/sh harbor.galasa.dev/galasadev/galasa-obr-with-galasabld:release`
-1. When inside the image, run `cd htdocs/dev/galasa`
-1. If the files maven-metadata.xml, maven-metadata.xml.md5 and maven-metadata.xml.sha1 are present, delete them, `rm maven-metadata.xml` etc.
-1. Go to Vault and find the maven-creds secret, to use the username and password in the next step.
-1. Navigate to the root directory in the image and then run the following command, to deploy all of the Maven artefacts we are releasing to the Nexus staging repository:
-`galasabld maven deploy --repository https://s01.oss.sonatype.org/service/local/staging/deploy/maven2 --local /usr/local/apache2/htdocs --group dev.galasa --version <GALASA_VERSION_WE_ARE_RELEASING> --username <USERNAME> --password <PASSWORD>`
-1. `exit` the image.
+1. Pull the [galasa-obr-with-galasabld](https://harbor.galasa.dev/harbor/projects/3/repositories/galasa-obr-with-galasabld/artifacts-tab) image from Harbor using:
+
+   ```
+   docker pull harbor.galasa.dev/galasadev/galasa-obr-with-galasabld:release
+   ```
+
+2. Exec into the image so you can run commands from inside it by running:
+
+   ```
+   docker run -it --entrypoint /bin/sh harbor.galasa.dev/galasadev/galasa-obr-with-galasabld:release
+   ```
+
+3. When inside the image, run `cd htdocs/dev/galasa`
+4. If the files maven-metadata.xml, maven-metadata.xml.md5 and maven-metadata.xml.sha1 are present, delete them, `rm maven-metadata.xml` etc.
+5. Go to Vault and find the maven-creds secret, to use the username and password in the next step.
+6. Navigate to the root directory in the image and then run the following command, to deploy all of the Maven artefacts we are releasing to the Nexus staging repository:
+
+   ```
+   galasabld maven deploy --repository https://s01.oss.sonatype.org/service/local/staging/deploy/maven2 --local /usr/local/apache2/htdocs --group dev.galasa --version <GALASA_VERSION_WE_ARE_RELEASING> --username <USERNAME> --password <PASSWORD>
+   ```
+
+7. `exit` the image.
 <!-- End of temporary steps -->
 1. 31-oss-sonatype-actions.md - Do the Sonatype actions detailed in this document.
```

Comment:
I think this could be a bit clearer as it goes from step 7 to a new set of instructions with no subtitle or something to describe what the next set of instructions are

Reply:
Addressed in https://github.com/galasa-dev/automation/pull/184/commits/1a8e361df1d826aac6bd01352740982abd570417